include/ajax.tasks.php

```diff
@@ -336,20 +336,18 @@ function massProcess($action, $w=null)  {
                 $info[':title'] = sprintf('Reopen %s',
                          _N('selected task', 'selected tasks', $count));
```

Comment:
@greezybacon
The problem with the longer phrases is that it is weird / wrong in German. 

To make it right the order of the words would needed to be switched, so e.g. "x selected Tickets Assign". As you can see it then would be weird / wrong in English.

Therefore I suggested to split up the phrase with  mdash, so there are 2 short phrases that are independently, but correct/right in both languages.

Don't know about other languages, but that is the issue with these phrases.

But  maybe you have a better suggestion apart from using mdash.

PS: Didn't you notice my earlier question about phrases with "try again"? ;)